[2025-05-21T16:06:09.686+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:06:09.731+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2025-05-21T16:06:09.796+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2025-05-21T16:06:09.805+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:06:09.889+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2022-10-03 00:00:00+00:00
[2025-05-21T16:06:09.900+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:06:09.904+0000] {standard_task_runner.py:72} INFO - Started process 3275 to run task
[2025-05-21T16:06:09.901+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2022-10-03T00:00:00+00:00', '--job-id', '1033', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmp74wfyibb']
[2025-05-21T16:06:09.915+0000] {standard_task_runner.py:105} INFO - Job 1033: Subtask get_name
[2025-05-21T16:06:10.091+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2022-10-03T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:06:10.512+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2022-10-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-03T00:00:00+00:00'
[2025-05-21T16:06:10.528+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:06:10.538+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2025-05-21T16:06:10.793+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:06:10.809+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2022-10-03T00:00:00+00:00, execution_date=20221003T000000, start_date=20250521T160609, end_date=20250521T160610
[2025-05-21T16:06:11.012+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:06:11.114+0000] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:06:11.123+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
